app/common/src/queryClient.ts

```diff
@@ -70,6 +73,18 @@ export interface QueryClientOptions<TStorageValue = string> {
   }
 }
 
+declare const brandRaw: unique symbol
+/**
+ * A value that is known not to be a reactive proxy; this marker type can be used to ensure
+ * comparisons are free of identity hazards.
+ */
+type RawValue<T> = T & { [brandRaw]: true }
```

Comment:
For brands, I've seen using `never` type instead of `true`. Which makes sense, because the value won't have `[brandRaw]` key.